addon/components/frost-link.js

```diff
@@ -58,6 +65,11 @@ export default Ember.LinkComponent.extend({
     addSizeClass(this.get('size'), classes)
     addPriorityClass(this.get('priority'), classes)
 
+    // primary link opens content in a new tab
+    if (classes.indexOf('primary') > -1) {
```

Comment:
Really minor, but I'd prefer to detect this off the point of entry (the priority attr)